resources/lang/de_DE/default.po

```diff
@@ -1187,8 +1187,14 @@ msgstr "Dauer"
 msgid "Name & Workmates"
 msgstr "Name & Kollegen"
 
-msgid "You have done enough."
-msgstr "Du hast genug gemacht."
+msgid "myshifts.sucess"
+msgstr "%s genug gemacht."
```

Comment:
That is some magical translation mushing which i can not recommend as it creates issues when only partial transslations get translated (and does not make sense for some other languages) -> please translate whole sentences / output groups at once!